geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -70,11 +70,27 @@ <h2 class="sr-only">
                 {% endif %}
             </dl>
             {# secondary metadata #}
-            {% if document.languages.exists or document.secondary_languages.exists %}
+            {% if document.languages.exists or document.secondary_languages.exists or document.doc_date_original or document.doc_date_standard %}
```

Comment:
do we really need this extra check? what's the downside to including an empty secondary metadata `dl` ?

Reply:
I was thinking about two things here: semantics and styling. If that's fine semantically, then that solves the former! Styling-wise, it's fine on desktop, but on mobile I was thinking we'd end up with this extra empty space.

<img width="311" alt="Screen Shot 2022-04-22 at 1 11 01 PM" src="https://user-images.githubusercontent.com/7234006/164763753-57a1c028-0490-46cc-b913-75c923af052c.png">

I think I came up with a solution to this though, which is just to use the `:first-child` CSS selector for that margin-top, instead of applying it to the entire `dl` 😄

Reply:
It seems ok to me semantically! Nothing will be announced to a screen-reader, right?

Could you use `:not(:empty)` ? (first child seems ok too, though)